src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,18 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
```

Comment:
I'm not sure if this is the best way to set up my makefile commands here. But if they are actually phony targets, should I put them in the `.PHONY` at top of file instead of in-line here?

Reply:
I need to remove the first `make` goal here as it already exists. And then I need to convert the other `make` goal from a `.PHONY` target to a real one.

Reply:
Moved away from .PHONY. closing